CHANGELOG.rst

```diff
@@ -4,10 +4,47 @@ Changelog
 This document describes changes between each past release.
 
 
-1.1 (unreleased)
+1.2 (2015-03-18)
 ----------------
 
-- Nothing changed yet.
+**Breaking changes**
+
+* `cliquet.storage.postgresql` now uses UUID as record primary key (#70)
+* Settings ``cliquet.session_backend`` and ``cliquet.session_url`` were
+  renamed ``cliquet.cache_backend`` and ``cliquet.cache_url`` respectively.
+* FxA user ids are not hashed anymore (#82)
+* Setting ``cliquet.retry_after`` was renamed ``cliquet.retry_after_seconds``
+* OAuth2 redirect url now requires to be listed in
+  ``fxa-oauth.webapp.authorized_domains`` (e.g. ``*.mozilla.com``)
+* Batch are now limited to 25 requests by default (#90)
+
+**New features**
+
+* Every setting can be specified via an environment variable
+  (e.g. ``cliquet.storage_url`` with ``CLIQUET_STORAGE_URL``)
+* Logging now relies on `structlog <http://structlog.org>`_ (#78)
+* Logging output can be configured to stream JSON (#78)
+* New cache backend for PostgreSQL (#44)
+* Documentation was improved on various aspects (#64, #86)
+* Handle every backend errors and return 503 errors (#21)
+* State verification for OAuth2 dance now expires after 1 hour (#83)
```

Comment:
issues/PR numbers won't match this repo. should we have a distinct paragraph for cliquet related stuff ?